### ISSUE TITLE
Investigate bad features

### ISSUE THREAD

**Issue by julianconstantine** [julianconstantine/Sberbank#8]
I have tried almost everything i can think of to prevent overfitting, and my cross-validation searches push me toward n_estimators=60, max_depth=2, and learning_rate=0.01, indicating a high degree of overfitting no matter what I do. However, other people on Kaggle using gradient boosting are (presumably) finding optimal hyperparameters that indicate a much greater degree of complexity in the underlying data.

Therefore, I suspect that some of the features I have created by hand are to blame.

**Comment by julianconstantine** [comment]
I already investigated dropping the "time series" features (indicating the number of sales per each month) and that did not alleviate the problem.

Currently, I am investigating the feature importance scores between the LightGBM starter script and the LightGBM run with my manually created features to look for any significant differences.

The most notable difference I have seen so far is that the variable age_at_sale and build_year are both significant, even though they basically indicate the same thing.
